<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationStepSelectorControls</class>
 <widget class="QWidget" name="RegistrationStepSelectorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pope View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    font: Roboto;
 /*   font: bold;*/
    font-size: 12px;
 /*   color: #FFF;*/
    background-color: transparent;
    border: 0px;
    padding: 6px;
	text-align: left
}
QPushButton:disabled {
   color: #595B5C;
}

QPushButton:hover
{
    background-color:rgba(0,0,0,20%);
}
QLabel
{
    font: Roboto;
/*    font: bold;*/
    font-size: 10pt;
/*    color: #FFF; */
}


QSpinBox, QSpinBox::before
{
    background-color: transparent;
    font: Roboto;
    font: normal;
    font-size: 10pt;
    color: #FFF; 
    border:0px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_ManualRegistration">
       <property name="text">
        <string>2. Manual Registration</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_AutomaticRegistration">
       <property name="text">
        <string>3. Automatic Registration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_RigidRegistration">
       <property name="styleSheet">
        <string notr="true">/*color: 'gray';*/</string>
       </property>
       <property name="text">
        <string>*5. Accurate registration</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_JuxtaposeResults">
       <property name="toolTip">
        <string>Visualize results by juxtaposing datasets</string>
       </property>
       <property name="styleSheet">
        <string notr="true">/*color: 'gray';*/</string>
       </property>
       <property name="text">
        <string>1. Image Fusion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_TimeFrameRegistration">
       <property name="text">
        <string>*4. Time Frame Registration</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_VisualizeResults">
       <property name="toolTip">
        <string>Visualize the correction made</string>
       </property>
       <property name="styleSheet">
        <string notr="true">/*color: 'gray';*/</string>
       </property>
       <property name="text">
        <string>*6. Visualize results</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
